harvesting-toolkit/README.md

```diff
@@ -137,21 +156,21 @@ During the process you may feel inclined to clean things up, add structure to th
 - Upload the Zip file using the application http://drp-upload.herokuapp.com/
    - Make sure to select the name of your event in the dropdown (and "remote" if you are working remotely)
    -  Note that files beyond 5 Gigs cannot be uploaded through this method
-     - Please talk to your DataRescue guide, if you have a larger file
- - Quality assurance:
-
+     - Please talk to your DataRescue guide/post on Slack in Researchers/Harvesters channel, if you have a larger file
```

Comment:
I was not sure if all the channels I mentioned already existed, and I thought some might need to be renamed to match our current nomenclature. Could somewhat check on that?